go.mod

```diff
@@ -1,12 +1,46 @@
 module github.com/ONSdigital/dp-reporter-client
 
-go 1.15
+go 1.20
```

Comment:
Should this be 1.20.4 instead? I'm not sure what difference it makes to be fair.

Reply:
But why do other repos specify 1.20.4 though? I'm just curious. E.g. on this page:

https://github.com/ONSdigital/dp-interactives-api/blob/feature/update-dp-kafka/ci/build.yml

Reply:
Sorry I thought I saw other repos pulling in version 1.20.4 - but I wasn't looking at the go.mod I realise. Thanks @redhug1